src/olympia/users/management/commands/clear_old_user_data.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime, timedelta
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+import olympia.core.logger
+from olympia import amo
+from olympia.addons.models import Addon
+from olympia.addons.tasks import delete_addons
+from olympia.users.models import UserProfile
+
+
+log = olympia.core.logger.getLogger('z.users')
+
+
+class Command(BaseCommand):
+    help = (
+        'Clear user data on addon developers deleted more than 7 years ago, '
+        'and non addon-developers after 24 hours')
+
+    def handle(self, *args, **options):
+        profile_clear = {
+            'last_login_ip': '',
+            'email': None,
+            'fxa_id': None,
+        }
+        one_day_ago = datetime.now() - timedelta(days=1)
+        seven_years_ago = datetime.now() - timedelta(days=365 * 7)
```

Comment:
It's not strictly 7 years, is that ok?

Reply:
7 years ish 😉 
@jvillalobos does it need to be exactly 7 years or is 365*7 close enough? (it's 1 or 2 days short depending on leap years)

Reply:
Close enough!